src/components/Forms/Subscribe.js

```diff
@@ -16,98 +16,140 @@ const SubscribeSchema = Yup.object().shape({
+                email_address: '',
+                first_name: '',
+              }}
+              validationSchema={SubscribeSchema}
+              onSubmit={values => this.handleSubmit(values)}
+              render={({ errors, touched }) => (
+                <Form
+                  css={css`
+                    display: flex;
+                    align-items: flex-end;
+                    label:not(:first-of-type),
+                    button {
+                      margin-left: 10px;
+                    }
+                    .field-error {
+                      display: block;
+                      position: absolute;
+                      color: ${theme.colors.red};
+                      font-size: 80%;
+                    }
+                    input,
+                    label {
+                      width: 280px;
+                    }
+                    ${bpMaxSM} {
+                      flex-direction: column;
+                      align-items: center;
+                      width: auto;
+                      label {
+                        margin: 15px 0 !important;
+                      }
+                      button {
+                        margin-top: 20px;
+                      }
+                    }
+                  `}
+                >
+                  <label htmlFor="first_name">
+                    First Name
+                    <Field
+                      aria-label="your first name"
+                      aria-required="true"
+                      name="first_name"
+                      placeholder="Jane"
+                      type="text"
+                    />
+                    <ErrorMessage
+                      name="first_name"
+                      component="span"
+                      className="field-error"
+                    />
+                  </label>
+                  <label htmlFor="email">
+                    Email
+                    <Field
+                      aria-label="your email address"
+                      aria-required="true"
+                      name="email_address"
+                      placeholder="jane@acme.com"
+                      type="email"
+                    />
+                    <ErrorMessage
+                      name="email_address"
+                      component="span"
+                      className="field-error"
+                    />
+                  </label>
+                  <button data-element="submit" type="submit">
+                    Submit
+                  </button>
+                  {/* <Debug /> */}
+                </Form>
+              )}
             />
-          </label>
-          <button data-element="submit" type="submit">
-            Submit
-          </button>
-          {/* <Debug /> */}
-        </Form>
-      )}
-    />
-  </div>
-)
+          )}
+        {submitted && !loading && <PostSubmissionMessage response={response} />}
+        {errorMessage && <div>{errorMessage}</div>}
```

Comment:
show the error message if there is one (which is set in the `catch` in `handleSubmit`